src/subcommand/server.rs

```diff
@@ -823,7 +823,7 @@ impl Server {
     );
     headers.insert(
       header::CONTENT_SECURITY_POLICY,
-      HeaderValue::from_static("default-src 'unsafe-eval' 'unsafe-inline' data:"),
+      HeaderValue::from_static("default-src 'self' 'unsafe-eval' 'unsafe-inline' data:"),
```

Comment:
can we limit the 'self' to /content/ ?

Reply:
see this comment: https://github.com/ordinals/ord/issues/1082#issuecomment-1500268793